magda-web-client/src/Components/RequestDataset/RequestDataset.js

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import RequestFormTemplate from "./RequestFormTemplate";
+import Alert from "./Alert";
+
+export default class RequestDataset extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            successResult: false,
+            posted: false
+        };
+    }
+    /**
+     * handles the logic of submitting form
+     * I still need the form submission logic to be implemented
+     * magda-dev.terria.io/api/v0/correspondence/send/dataset/request
+     * @param {Object} data form submitted data from child component "RequestFormTemplate.js"
+     */
+    handleSubmit = data => {
+        const senderEmail = data.senderEmail;
+        const message = data.message;
+        const senderName = data.senderName;
+        fetch(
```

Comment:
correspondenceApiURL should be defined in config https://github.com/TerriaJS/magda/blob/master/magda-web-client/src/config.js#L26